include/net/ethernet.h

```diff
@@ -628,18 +628,18 @@ static inline bool net_eth_get_vlan_status(struct net_if *iface)
  */
 #if defined(CONFIG_NET_VLAN)
 #define ETH_NET_DEVICE_INIT(dev_name, drv_name, init_fn, pm_control_fn,	\
-			    data, cfg_info, prio, api, mtu)		\
+			    data, cfg, prio, api, mtu)			\
```

Comment:
The macro was `config_info` here and in device.h, but this commit changes it to `cfg_ptr` in device.h but to `cfg` here.